dynamo/protocols/protocol_average_subtomograms.py

```diff
@@ -59,6 +59,12 @@ def _defineParams(self, form):
                       label='Do implicit rotation masking? (opt.)',
                       help='If set to Yes, in the rotated particles, the material outside a spherical mask will not '
                            'be computed. The particles will de facto appear with a spherical mask.')
+        form.addParam('randomizeOrientation', BooleanParam,
+                      default=True,
```

Comment:
As Dynamo provides oriented picking, I'd change the default to False to avoid losing the orientations.